README.md

```diff
@@ -39,8 +40,8 @@ describe get
 
 # Developers
 
-If you'd like to contribute, please see
-[CONTRIBUTING](https://github.com/knative/docs/blob/master/contributing/CONTRIBUTING.md)
+* If you'd like to contribute, please see
```

Comment:
links change ok, but lets not make them bullet points.

Reply:
:+1: